src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  let sexOfPartner = (
+    <p className="Person__partner">{`${partnerName} is my wife`}</p>
+  );
+
+  if (sex === 'f') {
+    sexOfPartner = (
+      <p className="Person__partner">{`${partnerName} is my husband`}</p>
+    );
+  }
```

Comment:
- It's better to create a separate function to get this text to avoid additional logic in the component
- 'f' is a magic value
- You repeat logic <p className="Person__partner">{`${partnerName} is my husband`}</p>

You can simplify like this
```
const personPartnerText = isMarried ?  `${partnerName} is my ${sex === SEX.female ? 'husband' : 'wife'}` : DEFAULT_PERSON_PARTNER_TEXT;
```